code/quarkus-telemetry/quarkus-telemetry-superhero/src/test/java/com/azure/examples/quarkus/superhero/message/SuperHeroCreatorTest.java

```diff
@@ -0,0 +1,36 @@
+package com.azure.examples.quarkus.superhero.message;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+
+@QuarkusTest
+class SuperHeroCreatorTest {
```

Comment:
We could perhaps remove the test folder to simplify the repo content.

Reply:
I'd prefer to keep the test, to make sure the code works... We should even add a native image test.